flutter_local_notifications/README.md

```diff
@@ -43,37 +43,208 @@ A cross platform plugin for displaying local notifications.
+}
+```
+
+Swift:
+```swift
+if #available(iOS 10.0, *) {
+  UNUserNotificationCenter.current().delegate = self as? UNUserNotificationCenterDelegate
+}
+```
+
+#### Handling notifications whilst the app is in the foreground
+By design, iOS applications *do not* display notifications the app is in the foreground. 
+
+For iOS 10+, use the presentation options to control the behaviour for when a notification is triggered while the app is in the foreground. 
+
+For older versions of iOS, you need to handle the callback as part of specifying the method that should be fired to the `onDidReceiveLocalNotification` argument when creating an instance `IOSInitializationSettings` object that is passed to the function for initializing the plugin. 
+
+Here is an example:
+
+```dart
+// initialise the plugin. app_icon needs to be a added as a drawable resource to the Android head project
+var initializationSettingsAndroid =
+    AndroidInitializationSettings('app_icon');
+var initializationSettingsIOS = IOSInitializationSettings(
+    onDidReceiveLocalNotification: onDidReceiveLocalNotification);
+var initializationSettings = InitializationSettings(
+    initializationSettingsAndroid, initializationSettingsIOS);
+flutterLocalNotificationsPlugin.initialize(initializationSettings,
+    onSelectNotification: onSelectNotification);
+
+...
+
+  Future onDidReceiveLocalNotification(
+      int id, String title, String body, String payload) async {
+    // display a dialog with the notification details, tap ok to go to another page
+    showDialog(
+      context: context,
+      builder: (BuildContext context) => CupertinoAlertDialog(
+            title: Text(title),
+            content: Text(body),
+            actions: [
+              CupertinoDialogAction(
+                isDefaultAction: true,
+                child: Text('Ok'),
+                onPressed: () async {
+                  Navigator.of(context, rootNavigator: true).pop();
+                  await Navigator.push(
+                    context,
+                    MaterialPageRoute(
+                      builder: (context) => SecondScreen(payload),
+                    ),
+                  );
+                },
+              )
+            ],
+          ),
+    );
+  }
+
+```
+
+
+
+## ❓ Usage
+Before going on to copy-paste the code snippets in this section, double-check you have configured your application correctly.
+If you encounter any issues please refer to the API docs and the sample code in the `example` directory before opening a request on Github.
+
+### Example
+Checkout `example` directory for a sample application that has been kindly implemented that demonstrate the features of this plugin.
+
+
+### API
+Checkout the lovely [pub-generated API documentation](https://pub.dev/documentation/flutter_local_notifications/latest/flutter_local_notifications/flutter_local_notifications-library.html). 
```

Comment:
@MaikuB any suggestions on how to word this sentence

Reply:
I'm ok with this, though if you're after alternatives the only one that comes to mind at the moment is "Checkout the lovely API documentation generated by pub"

Reply:
Done